src/lib.rs

```diff
@@ -271,6 +269,28 @@ impl AutoCfg {
         }
     }
 
+    /// Tests whether the given method can be used.
+    ///
+    /// The test code is subject to change, but currently looks like:
+    ///
+    /// ```ignore
+    /// pub static PROBE: SIG = METHOD;
```

Comment:
Hmm, this would actually work for any static declaration, where the `SIG` is a `Sync + 'static` type, and the initializer is any constant expression. That happens to be just fine for methods, but I think we should give this a more general name.

We could go further and allow *any* type and expression with a probe like:

```rust
pub fn probe() {
    let _: TYPE = EXPR;
}
```

Or in cases where we don't care to specify the exact type:

```rust
pub fn probe() {
    let _ = EXPR;
}
```

Reply:
Agree, more general expression will be better than a lot of `probe_XXX`.